debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -291,14 +290,18 @@ private PGReplicationStream startPgReplicationStream(final LogSequenceNumber lsn
     public synchronized void close() {
         try {
             super.close();
-        } catch (SQLException e) {
+        } catch (Exception e) {
```

Comment:
I think catching `Throwable` is better. I found many examples of this approach in Debezium project.

Reply:
I am not a fan of catching Throwables but in resource cleanup method it is justified.